api.js

```diff
@@ -0,0 +1,12 @@
+import axios from "axios";
+
+const newsApi = axios.create({
+	baseURL: `https://nc-news-4-life.onrender.com/api`,
+})
+
+export function fetchArticles() {
+    return newsApi.get(`/articles`)
+    .then((response) => {
+        return response.data
```

Comment:
I think it makes sense to just send the .articles here rather than needing to destructure it again once this function is invoked